### PR TITLE
tunneldriver: plumb the version through ngrok-go

### PR DESCRIPTION
**Pull request by jrobsonchase**
closes: #200 

## What

Surface the client type/version in the ngrok-go session.

## How

Pull in the latest version of ngrok-go and use its `WithChildClient`
option to set the client type/version for display in the ngrok
dashboard.

~~Also some plumbing via go:embed to get the version from the `VERSION`
file.~~

### CODE REVIEW

**Comment by jrobsonchase** [comment]
Current dependencies on/for this PR:
* main
  * **PR #227** <a href="https://app.graphite.dev/github/pr/ngrok/kubernetes-ingress-controller/227" target="_blank">Graphite</a> 
    * **PR #228** <a href="https://app.graphite.dev/github/pr/ngrok/kubernetes-ingress-controller/228" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ngrok/kubernetes-ingress-controller/228?utm_source=stack-comment).

**Review by nikolay-ngrok — APPROVED** [review]
👍

File: main.go [hunk 1]
```diff
@@ -56,6 +57,9 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+//go:embed VERSION
+var version string
+
```
Comment (alex-bezek) [inline main.go:62]: Oh this is really nifty. Do you think we can/should converge this with the other way we get the version via a build flag?
https://github.com/ngrok/kubernetes-ingress-controller/blob/main/internal/version/version.go#L11
https://github.com/ngrok/kubernetes-ingress-controller/blob/main/.github/workflows/docker.yaml#L51

Comment (jonstacks) [reply main.go:62]: 👍 to consolidating either way. My preference would be on the internal version package so we can get the git sha as well

Comment (jrobsonchase) [reply main.go]: Oops, my repo wasn't up to date! Explains why I didn't find this.

Moved the VERSION file to that package and added a symlink to the repo root, and replaced the ldflag with the embed.

**Review by jonstacks — CHANGES_REQUESTED** [review]
I think moving this will break some of the release automation.

**Comment by jrobsonchase** [comment]
@jonstacks Does any of that require `VERSION` to be a *file*? It's still where it was before - just as a symlink. `go:embed` has a hard requirement on it being a regular file beneath the current directory tree.

**Review by jonstacks — APPROVED** [review]

**Pull request merged**